.github/workflows/ci.yml

```diff
@@ -86,6 +86,26 @@ jobs:
       - name: Run prettier linter
         run: npm run checkformat
 
+  tests-jest:
+    name: Run Jest tests
+    runs-on: ubuntu-latest
+
+    steps:
+      - uses: actions/checkout@v4
+      - uses: actions/setup-node@v4
+        with:
+          node-version-file: '.nvmrc'
+          cache: npm
+
+      - name: Install dependencies
+        run: npm install
+
+      - name: Run tests
+        run: |
+          npm run test:jest
+        env:
+          CI: 'true'
```

Comment:
I thought this is detected automatically, is it not the case?

Reply:
I don't remember where I took this from